dev/dune_db

```diff
@@ -1,4 +1,27 @@
-source core_dune.dbg
+load_printer threads.cma
```

Comment:
I don't remember why we don't source core_dune.dbg, maybe it would work. OTOH maybe we hit some bug with source.

Reply:
According to `dune-build-system.md` because
> - OCaml >= 4.09 comes with `dynlink` already linked in so we need to
>   modify the list of modules loaded.

but we could probably just remove it from core_dune.dbg and source it

Reply:
Well, after testing, it seems you indeed were right with
> maybe we hit some bug with source

so I jut removed the source (dev/dune_db is now identical to former dev/dune_db_409 and seems to work properly)

Reply:
I think core_dune.dbg core.dbg base_db and db are all useless or broken now (we could rename dune_db to db)